train/compute/python/tools/execution_trace.py

```diff
@@ -298,6 +310,7 @@ def __init__(self, json):
             "1.0.2-chakra.0.0.4": ExecutionTrace._create_node_v1_0_2_chakra_0_0_4,
             # 1.0.3 expands pg name to <pg_name, pg_desc> so it use the same parser as 1.0.2
             "1.0.3-chakra.0.0.4": ExecutionTrace._create_node_v1_0_2_chakra_0_0_4,
+            "1.1.0-chakra.0.0.4": ExecutionTrace._create_node_v1_0_2_chakra_0_0_4,
```

Comment:
I bumped it to 1.0.4-chakra.0.0.4. Looks like you have not merged my PR yet. 

Why bump to 1.1.0, skipping other numbers? Is it because it is a important change?

Reply:
Yea It would take sometime for my change to land and its a significant one